katacomb/katacomb/uv_facade.py

```diff
@@ -401,6 +401,26 @@ def Desc(self):
     def List(self):
         return self.uv.List
 
+    @property
+    def nvis_from_NX(self):
+        """ Derive the number of vis from the attached NX table
+
+            NOTE: Sometimes in certain corner cases the 'nvis'
```

Comment:
Docstring should only be quoted as much as the opening quotes ([PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)) - but again, if this is a convention already in use in this file, don't worry trying to fix it now.

Reply:
Yes. Done.